book.rb

```diff
@@ -0,0 +1,15 @@
+require './rental'
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  attr_accessor :title, :author
+  attr_reader :rentals
+
```

Comment:
- [ ] Since rentals will be created for books, the getters and setters for rentals should be present.